modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -67,25 +67,22 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 		Locale displayLocale = LocaleUtil.fromLanguageId(languageId);
 
 		DDMFormInstance formInstance = ddmFormDisplayContext.getFormInstance();
+
+		boolean showSuccessPage = ddmFormDisplayContext.isShowSuccessPage();
+
+		boolean preview = ddmFormDisplayContext.isPreview();
```

Comment:
@rodrigopaulino is this blank line necessary for a reason or can we order alphabetically?

Reply:
Not necessary. I just figured I should order it by their usage. Please, let me know what's the correct way.

Reply:
We could do `(ddmFormDisplayContext.hasSubmittedAnEntry() && !preview) || showSuccessPage` in this case.

Reply:
I think I'll have to reformulate this clause anyway, because I'm seeing that you can't preview forms that are expired and I believe this shouldn't be a criteria.

Reply:
I'll talk to Luiz about it.

Reply:
Ok! Thanks, @rodrigopaulino.